monkey/monkey_island/cc/setup/island_config_options.py

```diff
@@ -20,22 +19,39 @@
 
 
 class IslandConfigOptions:
-    def __init__(self, config_contents: dict = None):
-        if not config_contents:
-            config_contents = {}
-        self.data_dir = config_contents.get(_DATA_DIR, DEFAULT_DATA_DIR)
+    def __init__(self, config_contents: dict = {}):
+        self.data_dir = None
+        self.log_level = None
+        self.start_mongodb = None
+        self.crt_path = None
+        self.key_path = None
+
+        self._initialize_defaults()
+        self.update(config_contents)
+
+    def _initialize_defaults(self):
+        self.data_dir = DEFAULT_DATA_DIR
+        self.log_level = DEFAULT_LOG_LEVEL
+        self.start_mongodb = DEFAULT_START_MONGO_DB
+        self.crt_path = DEFAULT_CRT_PATH
+        self.key_path = DEFAULT_KEY_PATH
```

Comment:
Why not just set these in __init__?

Reply:
I can make that change. See https://github.com/guardicore/monkey/pull/1668/files#r781174189 for an explaination.